app/controllers/application_controller.rb

```diff
@@ -1000,6 +1000,7 @@ def get_sort_col
       @sortcol = 0
       @sortdir = "ASC"
     end
+    params[:is_ascending] = @sortdir.to_s.downcase != "desc"
     @sortdir = params[:is_ascending] ? 'ASC' : 'DESC' unless params[:is_ascending].nil?
```

Comment:
`params[:is_ascending]` can no longer be nil after this change, so this can simplify to

```suggestion
    params[:is_ascending] = @sortdir.to_s.downcase != "desc"
    @sortdir = params[:is_ascending] ? 'ASC' : 'DESC'
```

Reply:
Actually now that I'm reading this close, won't this break line 991-995 where it sets the sort dir?

Reply:
It seems to be working fine